docs/UserGuide.adoc

```diff
@@ -139,7 +139,7 @@ The `undo` command fails as there are no undoable commands executed previously.
 ==== Adding a member: `memberAdd`
 
 You can add members to ClubManager. +
-Format: `memberAdd n/NAME an/ADMIN_NUMBER p/PHONE_NUMBER e/EMAIL yos/YEAR_OF_STUDY m/MAJOR [t/TAG]...`
+Format: `memberAdd n/NAME an/MATRIC_NUMBER p/PHONE_NUMBER e/EMAIL yos/YEAR_OF_STUDY m/MAJOR [t/TAG]...`
```

Comment:
May want to change the matric number tag to mn instead of an

Reply:
yup done!